testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/ejb/timerservice/attribute/TimerAttributeDefinitionTestCase.java

```diff
@@ -0,0 +1,68 @@
+package org.jboss.as.test.integration.ejb.timerservice.attribute;
+
+import org.jboss.as.controller.SimpleOperationDefinitionBuilder;
+import org.jboss.as.controller.descriptions.StandardResourceDescriptionResolver;
+import org.jboss.as.ejb3.subsystem.deployment.TimerAttributeDefinition;
+
+import org.jboss.dmr.ModelNode;
+import org.jboss.dmr.ModelType;
+import org.junit.Test;
+
+import java.util.Locale;
+
+import static org.jboss.as.controller.SimpleAttributeDefinitionBuilder.create;
+import static org.jboss.as.controller.descriptions.ModelDescriptionConstants.DESCRIPTION;
+import static org.jboss.as.controller.descriptions.ModelDescriptionConstants.NILLABLE;
+import static org.jboss.as.controller.descriptions.ModelDescriptionConstants.REPLY_PROPERTIES;
+import static org.jboss.as.controller.descriptions.ModelDescriptionConstants.TYPE;
+import static org.jboss.as.controller.descriptions.ModelDescriptionConstants.UNIT;
+import static org.jboss.as.controller.descriptions.ModelDescriptionConstants.VALUE_TYPE;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+public class TimerAttributeDefinitionTestCase {
```

Comment:
@petrberan This test should be in the ejb3 maven module.

Reply:
Thank you, moved